src/System.Management.Automation/engine/CommandParameter.cs

```diff
@@ -15,12 +15,14 @@ internal sealed class CommandParameterInternal
     {
         private class Parameter
         {
+            internal Ast ast;
             internal IScriptExtent extent;
```

Comment:
I mean - `extent` should always be `ast.Extent`, so we could remove the field and use a property instead.